### PR TITLE
[NIFI-12504] Remaining views on Summary page

### PR DESCRIPTION
**Pull request by rfellows**
# Summary

[NIFI-12504](https://issues.apache.org/jira/browse/NIFI-12504)

* process group summary tab
* ellipsify data/headers in table
* input and output port summary tabs
* connections summary tab
* rpg summary tab

### CODE REVIEW

**Comment by mcgilman** [comment]
Will review...

**Review by mcgilman — CHANGES_REQUESTED** [review]
Thanks for the PR @rfellows!

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/summary/ui/common/port-status-table/port-status-table.component.ts [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+
+    get initialSortDirection() {
+        return this._initialSortDirection;
+    }
+
+    @Input() set ports(ports: PortStatusSnapshotEntity[]) {
+        if (ports) {
+            this.dataSource.data = this.sortEntities(ports, this.multiSort);
+            this.dataSource.filterPredicate = (data: PortStatusSnapshotEntity, filter: string) => {
+                const filterArray: string[] = filter.split('|');
```
[the remark this answers is not in the document]
Comment (rfellows) [reply nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/summary/ui/common/port-status-table/port-status-table.component.ts:102]: Good catch. I'll come up with a solution

**Review by mcgilman — APPROVED** [review]
Thanks for the updates @rfellows! Will merge once CI completes. +!

**Comment by rfellows** [comment]
recheck

**Pull request merged**